tests/integration/test_runner_manager_openstack.py

```diff
@@ -142,6 +143,8 @@ async def openstack_runner_manager_fixture(
         server_config=server_config,
         runner_config=runner_config,
         service_config=service_config,
+        # we assume the test runs as ubuntu user
+        system_user_config=SystemUserConfig(user="ubuntu", group="ubuntu"),
```

Comment:
This assumes that this test is run on the CI as a ubuntu user. I assume our tests are not intended to be run locally, so this should be fine.
An alternative would be to do more patching of internas (with the downside of not "really" doing an integration test).